src/test/java/test/appium_tests/AppiumTest.java

```diff
@@ -0,0 +1,63 @@
+package test.appium_tests;
+
+import io.appium.java_client.MobileElement;
+import io.appium.java_client.android.AndroidDriver;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.remote.CapabilityType;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import pages.CalculatorPage;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+@RunWith(JUnitParamsRunner.class)
+public class AppiumTest {
+
+    protected AndroidDriver<MobileElement> driver;
+
+    @Before
+    public void setUp() throws MalformedURLException {
```

Comment:
Update readme to clarify which steps need to do to be able to run this test

Reply:
partialy done